lib/tasks/one_off/create_contract.rake

```diff
@@ -0,0 +1,54 @@
+COHORT_YEAR = 2023
+STATEMENT_YEAR = 2025
+STATEMENT_MONTH = 1
+LEAD_PROVIDER_NAME = "LLSE".freeze
```

Comment:
This will end up in the global namespace for anything reading the rake task file

I think its fine for there to either be constants or variables in the rake task

Reply:
that's a very good point - I've wrapped the rake task definition in a class - so those constants should not leak into the global namespace now.